Milvus.Client.Tests/DataTests.cs

```diff
@@ -113,15 +113,18 @@ public async Task Timestamp_conversion()
     [Fact]
     public async Task Flush()
     {
-        await Collection.WaitForFlushAsync();
         // Any insertion after a flush operation results in generating new segments.
         await InsertDataAsync(5, 6);
+        // Wait to avoid rate limiting.
```

Comment:
Is this really necessary when testing against a local container? It makes the development experience pretty bad if the developer has to sit around and wait for 12 seconds etc. Maybe there's a way to just disable rate limiting on the container or similar?

If not, then I guess if it's really just the Flush tests, maybe it's OK...

Reply:
I was able to disable the rate limit and I removed the delays.